exercises/sieve/src/sieve.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _SIEVE_H
+#define _SIEVE_H
+
+#define MAX_LIMIT_TESTED (1000)
+
+typedef unsigned int primesArray_t[MAX_LIMIT_TESTED];
```

Comment:
It would make the testing more annoying, but it seems like a callback solution would be more natural and efficient